### PR TITLE
add flappybird game

### PR DESCRIPTION
**Pull request by tigercoding56**
add flappy bird game
here is a screenshot
image

### CODE REVIEW

File: apps/flappybird.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+from micropython import const
+import wasp
+import drivers 
+import micropython
+import machine
+import ble
+import time
+print(dir(ble))
+
+@micropython.native
```
Comment (fgaz) [inline apps/flappybird.py:12]: is `native` really needed? space is precious

File: apps/flappybird.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: MY-LICENSE
+# Copyright (C) YEAR(S), AUTHOR
```
[the remark this answers is not in the document]
Comment (tigercoding56) [reply apps/flappybird.py:2]: ok

File: apps/flappybird.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+from micropython import const
+import wasp
+import drivers 
+import micropython
+import machine
+import ble
+import time
+print(dir(ble))
```
[the remark this answers is not in the document]
Comment (tigercoding56) [reply apps/flappybird.py:10]: ups

File: apps/flappybird.py (hunk 1 above)
Comment (tigercoding56) [reply apps/flappybird.py:12]: well originally the game  had better graphics (and needed it for performance ) , after some testing i found that performance stayed almost the same  though (now that it has a simpler design) 'ill remove it

File: docs/apps.rst [hunk 4]
```diff
@@ -81,3 +81,5 @@ Games
 .. automodule:: apps.play2048
 
 .. automodule:: apps.snake
+
+.. automodule:: apps.flappybird
```
Comment (fgaz) [inline docs/apps.rst:85]: It's an optional app, so it should be `Flappybird` instead of `apps.flappybird` due to how app discovery works.

**Review by fgaz — CHANGES_REQUESTED** [review]
You should also add the app to the README file

File: docs/apps.rst (hunk 4 above)
Comment (tigercoding56) [reply docs/apps.rst:85]: ok Ty

**Review by fgaz — CHANGES_REQUESTED** [review]
rename `apps/flappybird.py` to `apps/Flappybird.py`

**Comment by tigercoding56** [comment]
> rename `apps/flappybird.py` to `apps/Flappybird.py`

 ok

**Comment by fgaz** [comment]
Same as https://github.com/wasp-os/wasp-os/pull/401#issuecomment-1450560027

**Comment by tigercoding56** [comment]
> Same as [#401 (comment)](https://github.com/wasp-os/wasp-os/pull/401#issuecomment-1450560027)

no , or did i mess up my branches again?

File: README.rst [hunk 5]
```diff
@@ -265,3 +265,7 @@ application (and the "blank" white screen is a torch application):
 .. image:: res/BeaconApp.png
    :alt: Flash the relatively powerful HRS LED repeatedly
    :width: 179
+
+.. image:: res/FlappyApp.png
```
Comment (fgaz) [inline README.rst:269]: It has to be consistent with everything else, so `res/FlappybirdApp.png`, also rename the file

**Comment by beardeddude** [comment]
As you may know after the merge of #403 the naming conventions and file placement have changed slightly. See https://wasp-os.readthedocs.io/en/latest/appguide.html#app-naming-conventions-and-placement for details.

**Comment by tigercoding56** [comment]
I'll close this as school started again and i do not have as much time as i had during break if anyone wants to use this application on their own versions of wasp-os go ahead

**Pull request closed without merging**